activities/Markdown.activity/js/activity.js

```diff
@@ -1,13 +1,17 @@
-define(["sugar-web/activity/activity","sugar-web/datastore"], function (activity, datastore) {
+define(["sugar-web/activity/activity","webL10n","sugar-web/datastore"], function (activity, datastore) {
 
     // Manipulate the DOM only when it is ready.
     require(['domReady!'], function (doc) {
 
         // Initialize the activity.
         activity.setup();
-
+		var buttons = ["wmd-bold-button-second", "wmd-italic-button-second", "wmd-heading-button", "wmd-hr-button",
+			       "wmd-olist-button", "wmd-ulist-button", "wmd-code-button", "wmd-quote-button", "wmd-link-button",
+				"wmd-undo-button", "wmd-redo-button"];
         inputTextContent = document.getElementById("wmd-input-second");
-        inputTextContent.value = "#This is a sample input";
+        inputTextContent.value = l10n_s.get("sample-input");
+		//document.getElementById("wmd-bold-button-second");
```

Comment:
I expect for each id (in your array) something like:
document.getElementById("wm-bold-button-second".title = l10n_s.get("wmd-bold-button-second");